allennlp/data/dataset_readers/language_modeling.py

```diff
@@ -2,7 +2,7 @@
 
 from overrides import overrides
 
-from . import DatasetReader
+from .dataset_reader import DatasetReader
```

Comment:
Would absolute imports have fixed this?  Also, in general, when we touch code, we should probably switch it to absolute imports.  Or we could make a PR that just does it all in one go, but that's more work...